### PR TITLE
Update data-plane-api

### PR DESCRIPTION
**Pull request by kyessenov**
- introduce a few well-known names for filters

Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: pkg/cache/resource.go [hunk 1]
```diff
@@ -30,10 +30,51 @@ type Resource interface {
...
+
+	// Echo network filter
+	Echo = "envoy.echo"
+	// HTTPConnectionManager network filter
 	HTTPConnectionManager = "envoy.http_connection_manager"
-	TCPProxy              = "envoy.tcp_proxy"
+	// TCPProxy network filter
+	TCPProxy = "envoy.tcp_proxy"
+	// RateLimit network filter
+	RateLimit = "envoy.ratelimit"
```
Comment (kyessenov) [inline pkg/cache/resource.go]: Can you please double-check this list?

File: pkg/cache/resource.go [hunk 2]
```diff
@@ -30,10 +30,51 @@ type Resource interface {
 
 // Common names for Envoy filters.
 const (
-	CORS                  = "envoy.cors"
-	Router                = "envoy.router"
+	// Buffer HTTP filter
+	Buffer = "envoy.buffer"
```
Comment (9len) [inline pkg/cache/resource.go]: is `cache` the right package for these?

Comment (kyessenov) [reply pkg/cache/resource.go]: It's not ideal. We have "cache", "server", "util", "test", "log" packages. Which one would be better? Should we create a new one "filter", maybe?

Comment (kyessenov) [reply pkg/cache/resource.go]: cc @ostromart since he recently touched this code

Comment (9len) [reply pkg/cache/resource.go]: I would say either util or filter.

Comment (kyessenov) [reply pkg/cache/resource.go]: In the interest of package economy, I'll move these well-known names to util.

**Pull request merged**